pyproject.toml

```diff
@@ -22,7 +22,7 @@ requires = [
 
     # numpy 1.19 was the first minor release to provide aarch64 wheels, but
     # wheels require fixes contained in numpy 1.19.2
-    "numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64'",
+    "numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'",
```

Comment:
So if I'm reading the discussions correctly, `master` branch actually isn't broken for aarch64 + PyPy but this is "good to have?"

In the other repo it says:

> no conflict on scipy master, but better to also sync this info
 
Worth backporting to 1.8.x series as well, or not really?

Reply:
Yes, it's no conflict, but force user to install `numpy==1.19.2` with pypy3 aarch64
- before this patch, the **numpy v1.19.2** would be installed with pypy3 aarch64 (but no wheel provided, still install with source in pip)
- after this patch, the **numpy lastest** would be installed with pypy3 aarch64